src/sentry/api/event_search.py

```diff
@@ -913,7 +913,21 @@ def validate_aggregate(field, match):
             },
         ],
         "transform": "quantile(%(percentile).2f)(%(column)s)",
-    }
+    },
+    "rps": {
+        "name": "rps",
+        "args": [
+            {"name": "interval", "type": NUMBER, "validator": lambda v: (v > 0, "must be positive integer")},
+        ],
+        "transform": "divide(count(), %(interval)d)",
+    },
+    "rpm": {
+        "name": "rpm",
+        "args": [
+            {"name": "interval", "type": NUMBER, "validator": lambda v: (v > 0, "must be positive integer")},
+        ],
+        "transform": "divide(count(), divide(%(interval)d, 60))",
```

Comment:
Is the plan to move these into snuba once the AST is farther along?

Reply:
Yep.

Reply:
can interval ever be smaller than a minute ?

Reply:
Yea it can.

Reply:
Then I guess the result of rpm will not be really meaningful ?

Reply:
Unfortunately not. However, I believe from the frontend on product side, we don't go below 1 minute granularity.

Reply:
We could disallow intervals smaller than a minute in the function validator.

Reply:
Yeah I can add validation for that.